README.md

```diff
@@ -11,21 +11,69 @@ See https://mercure.rocks.
 Make sure you change in `.platform/applications.yaml`
 
 ```
-JWT_KEY: "!ChangeMe!"
-DEMO: 1
+MERCURE_PUBLISHER_JWT_KEY='!ChangeMe!'
+MERCURE_SUBSCRIBER_JWT_KEY='!ChangeMe!'
 ```
 
-You should also remove the `public` directory, that is not needed for production.
+You probably also want to disable the demo/ui in:
+`mercure/Caddyfile.platform_sh` and possibly allowing anonymous users.
```

Comment:
Does `Caddyfile.platform_sh` file different from the default `Caddyfile.dev` one? If not, why not use the default? Then the maintenance of that file will be on Mercure devs